src/jwplayer/players/MobileArticleVideoPlayer/MobileArticleVideoPlayer.tsx

```diff
@@ -76,9 +77,12 @@ export const MobileArticleVideoPlayerContent: React.FC<MobileArticleVideoPlayerP
 
 	useEffect(() => {
 		if (!onScreen) {
+			relatedContainer?.classList.add('hidden');
```

Comment:
Maybe another way to achieve our goal is to close the related plugin when it's not in the viewport 🤔 https://docs.jwplayer.com/players/reference/related

Reply:
Yes, but I guess we want to maintain the _Related Videos_ feature.